src/wing/CCPACSWing.h

```diff
@@ -64,125 +77,250 @@ friend class CTiglWingBuilder;
+    /**
+     * @brief Returns the segment for a given index or uid
+     * @param index Segment index
+     * @param uid Segment UID
+     * @return CCPACSWingSegment&
+     */
     TIGL_EXPORT CCPACSWingSegment& GetSegment(const int index);
     TIGL_EXPORT const CCPACSWingSegment& GetSegment(const int index) const;
     TIGL_EXPORT CCPACSWingSegment& GetSegment(std::string uid);
     TIGL_EXPORT const CCPACSWingSegment& GetSegment(std::string uid) const;
 
-    // Getter of the number of component segments of the wing
+    /**
+     * @brief Returns the number of component segments of the wing
+     * @return int
+     */
     TIGL_EXPORT int GetComponentSegmentCount() const;
 
-    // Returns the segment for a given index or uid
+    /**
+     * @brief Returns the segment for a given index or uid
+     * @param index Segment index
+     * @param uid Segment UID
+     * @return CCPACSWingComponentSegment&
+     */
     TIGL_EXPORT CCPACSWingComponentSegment& GetComponentSegment(const int index);
     TIGL_EXPORT const CCPACSWingComponentSegment& GetComponentSegment(const int index) const;
     TIGL_EXPORT CCPACSWingComponentSegment& GetComponentSegment(const std::string& uid);
 
-    // Get the positioning transformation for a given section uid
+    /**
+     * @brief Returns the positioning transformation for a given section uid
+     * @param sectionUID
+     * @return CTiglTransformation
+     */
     TIGL_EXPORT CTiglTransformation GetPositioningTransformation(std::string sectionUID);
 
-    // Gets the upper point in absolute (world) coordinates for a given segment, eta, xsi
+    /**
+     * @brief Returns the upper point in absolute (world) coordinates for a given segment, eta, xsi
+     * @param segmentIndex
+     * @param eta
+     * @param xsi
+     * @return gp_nP
+     */
     TIGL_EXPORT gp_Pnt GetUpperPoint(int segmentIndex, double eta, double xsi);
 
-    // Gets the upper point in absolute (world) coordinates for a given segment, eta, xsi
+    /**
+     * @brief Returns the lower point in absolute (world) coordinates for a given segment, eta, xsi
+     * @param segmentIndex
+     * @param eta
+     * @param xsi
+     * @returns gp_nP
+     */
     TIGL_EXPORT gp_Pnt GetLowerPoint(int segmentIndex, double eta, double xsi);
 
-    // Gets a point on the chord surface in absolute (world) coordinates for a given segment, eta, xsi
+    /**
+     * @brief Returns a point on the chord surface in absolute (world) coordinates for a given segment, eta, xsi
+     * @param segmentIndex
+     * @param eta
+     * @param xsi
+     * @param referenceCS
+     * @returns gp_nP
+     */
     TIGL_EXPORT gp_Pnt GetChordPoint(int segmentIndex, double eta, double xsi, TiglCoordinateSystem referenceCS = GLOBAL_COORDINATE_SYSTEM);
 
-    // Gets the loft of the whole wing
+    /**
+     * @brief Gets the loft of the whole wing
+     * @return TopoDS_Shape& Returns a reference to the OCC Type TopoDS_Shape
```

Comment:
I am not sure I would understand the description "Returns a reference to the OCC Type TopoDS_Shape".

Reply:
Yes, I'll try to keep it simple. I still think, it might be helpful to point out, that a TopoDS_Shape is part of OCC, and that this function uses the configuration to provide a geometric object?